plugins/callback/influxdb.py

```diff
@@ -0,0 +1,150 @@
+# (c) 2019, Jason Neurohr, <jason@jasonneurohr.com>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: influxdb
+    callback_type: aggregate
+    requirements:
+      - whitelist in configuration
+    short_description: Adds play duration to InfluxDB
+    version_added: "1.3.0"
+    description:
+        - This callback captures the total play duration and submits it to InfluxDB.
+    options:
+      influxdb_addr:
+        required: False
+        description: InfluxDB address in the form FQDN, FQDN:PORT, IP, or IP:PORT
```

Comment:
```suggestion
        description: "InfluxDB address in the form FQDN, FQDN:PORT, IP, or IP:PORT."
```
when ':' is in a line, we must double-quoted the line